apps/prairielearn/src/tests/groupExamRolePermissions.test.sql

```diff
@@ -0,0 +1,59 @@
+-- BLOCK select_assessment
+SELECT
+  a.id
+FROM
+  assessments AS a
+WHERE
+  a.tid = $assessment_tid;
+
+-- BLOCK generate_and_enroll_3_users
+SELECT
+  user_id,
+  uid,
+  name,
+  uin
+FROM
+  users_randomly_generate (3, 1)
+  LEFT JOIN course_instances AS ci on (ci.id = 1)
+  LEFT JOIN pl_courses AS c ON (c.id = ci.course_id)
```

Comment:
What do these joins do for us?

Reply:
I copied this from #8006 and didn't change it. The same join is used in other tests. It seems this was blindly copied from the admin query that generates random users (which uses these joins) by one of the tests, then copied from one test to another without thinking. I removed this (and other related joins) in b544f3d67.